contracts/Example3.sol

```diff
@@ -7,8 +7,10 @@ contract Example3 {
         view
         returns (uint256 amountForSender, uint256 amountForOwner)
     {
-        amountForSender = (amount * (100 - marginPercentage)) / 100;
-        amountForOwner = (amount * marginPercentage) / 100;
+        uint _marginPercentage = marginPercentage;
+        uint _amount=amount;        
```

Comment:
You don't need to store `amount` into another variable, it is already present in memory